osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -270,26 +269,34 @@ - (enum kaliteStatus)checkRunTask:(NSNotification *)aNotification{
     if (!isKaliteCommand(lastCommandArg)) {
         if (status == statusOkRunning) {
             self.portAlertCounter += 1;
```

Comment:
This is confusing me, if we are just checking if a port is used, then why use increment/decrement when a simple Boolean value will do?  Any particular reason for this?